### ISSUE TITLE
Hard crash of 32bit NPP when formatting a single line of JSON

### ISSUE THREAD

**Issue by boris-v** [sunjw/jstoolnpp#152]
Windows, NPP latest version, 32bit. Latest plugin version (1.2312).
The bug is only with 32bit NPP; not reproducible on 64bit npp (same version of NPP & plugin).

Test JSON line:
```
[{"i": null, "j": null, "k": null, "l": "P1", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": null, "l": "P2", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": null, "l": "P3", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": null, "l": "P4", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": null, "l": "P5", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": "qwerty", "k": "zxcvb", "l": "P1", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": "qwerty", "k": "zxcvb", "l": "P2", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": "qwerty", "k": "zxcvb", "l": "P3", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": "qwerty", "k": "zxcvb", "l": "P4", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": "qwerty", "k": "zxcvb", "l": "P5", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": "zxcvb", "l": "P1", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": "zxcvb", "l": "P2", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": "zxcvb", "l": "P3", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": "zxcvb", "l": "P4", "m": false, "n": "a", "o": "szdf", "p": 123456}, {"i": null, "j": null, "k": "zxcvb", "l": "P5", "m": false, "n": "a", "o": "szdf", "p": 123456}]
```

1. Paste the JSON into a blank npp document, single line.
![image](https://github.com/sunjw/jstoolnpp/assets/3235397/1a0904e0-7c10-485f-bbdf-0a4385868d19)

- Plugin JSTool -> JSFormat: formats JSON
(Works correctly)
![image](https://github.com/sunjw/jstoolnpp/assets/3235397/4d6ab5d0-85db-41ca-bb7d-06669d393d1e)

- Select all, with the entire line selected (but it must be a single line):
![image](https://github.com/sunjw/jstoolnpp/assets/3235397/56e10be3-68a1-49f9-a8d7-c708bbd661e9)
apply JSFormat: Causes NPP to crash hard

2. It works correctly with the selection if a single newline is added to the end.
Paste the JSON into a blank document, single line, add one newline (Unix - LF), Select All. 
![image](https://github.com/sunjw/jstoolnpp/assets/3235397/5a3fa0a0-5741-4b64-89c5-8ee9f537bf9f)
Plugin JSTool -> JSFormat with this JSON line + newline selected:
![image](https://github.com/sunjw/jstoolnpp/assets/3235397/9c07dd76-5d78-4be9-82ec-2585804a5b0e)

**Comment by sunjw** [comment]
I couldn't reproduce this issue. You can see the screen record I attached. Is there any step wrong in the record?

https://github.com/sunjw/jstoolnpp/assets/846335/f11f681e-81d4-4a2e-8b09-c5091d047b92

**Comment by boris-v** [comment]
Thanks for looking into it!
I cannot reproduce it anymore, either 😓 

I only realized I still had the 32bit version when making this issue, and I replaced it with 64bit afterwards. I now reinstalled the 32bit version, installed the plugins, and could not reproduce it again.

Anything more is just guessing... It might be that something went wrong with one of the updates, I've had that 32bit installation for many years and I just kept upgrading NPP and the plugins.
